pkg/storage/unified/search/testdata/doc/user-example-out.json

```diff
@@ -0,0 +1,17 @@
+{
+  "key": {
+    "namespace": "default",
+    "group": "iam.grafana.app",
+    "resource": "users",
+    "name": "example"
+  },
+  "name": "example",
+  "rv": 1234,
+  "title": "example",
+  "title_ngram": "example",
+  "title_phrase": "example",
```

Comment:
do we want title to be the k8s name, or more like todays "displayName" login, email, or name